src/poke_env/environment/target.py

```diff
@@ -0,0 +1,57 @@
+"""This module defines the Target class, which represents the types of
+targets a move can have
+"""
+
+import logging
+from enum import Enum, unique, auto
+
+# This is an enum for all the targets you can have
+@unique
+class Target(Enum):
```

Comment:
i think i'd prefer `ADJACENT_ALLY` style names to `ADJACENTALLY` - they are a but more readable

Reply:
Sounds good - will convert!